### DISCUSSION TITLE
Is this the version 1.25.2 the same as what I get with :latest

### DISCUSSION THREAD

**Discussion by Docfxit** [teslamate-org/teslamate#2737] [Getting started]
On this page about half way down it has a post from adriankumpf saying:
[v1.25.2](https://github.com/adriankumpf/teslamate/releases/tag/v1.25.2) **_is out with a fix for this error._**

Is that the same version I get when I use:
image: teslamate/teslamate:latest
In my docker-compose.yml file?
Or is it a correction to the 1.25.2 file?

**Comment by cwanja** [comment]
What page, referring back to your opening statement? 

Using latest in your Docker file should get you `1.26.1`

**Reply by Docfxit** [reply]
> 

I am currently running 1.26.1
My question is about 1.26.2.
You can find it here:
https://github.com/adriankumpf/teslamate/releases/tag/v1.25.2

**Reply by cwanja** [reply]
You keep referencing 1.**25**.x releases. Both in your original post and this response, a link to release 1.**25**.2. Currently there is no 1.**26**.2. 

Otherwise, I am not sure what you are asking for or about.
